tsconfig.json

```diff
@@ -5,8 +5,11 @@
     "**/build/**/*"
   ],
   "compilerOptions": {
+    "jsx": "react",
```

Comment:
This actually should have no impact at all since we build via Babel.

Reply:
I think I added it just to make my IDE happy (WebStorm/Idea). Probably can find a way to do without.

Reply:
No, it is fine, doesn't do any harm. (The original intent is just lost, since it effectively has no impact :))